.gitignore

```diff
@@ -5,4 +5,5 @@
 .*.swp
 *.vim
 /runspike.sh
+/runspiketest.sh
```

Comment:
Can we keep these sorts of arbitrary filenames out of the .gitignore?  Otherwise it'll end up having every random script anyone ever writes.  You can add them to your local `.git/info/exclude` file instead.

Reply:
Cool, thanks! I didn't know before that there were other ways of ignoring files with git.